.github/workflows/SBOM.yaml

```
@@ -0,0 +1,35 @@
+name: Generate SBOM
+
+on:
+  push:
+    
+
+jobs:
+  generate_sbom:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2  #get depo code
```

No need for a comment here
```suggestion
        uses: actions/checkout@v2
```